scss/tech.scss

```diff
@@ -0,0 +1,42 @@
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-flow: column wrap;
+  margin: 15px;
+  padding: 10px;
+  position: relative;
+  text-align: center;
+  width: 190px;
+
+  &::before {
+    border-radius: 5px;
+    border: 1px solid $green;
+    content: '';
+    height: 241px;
```

Comment:
try to use even numbers if possible, they are easier to work with, use 240 (or 250 if 240 is not enough) for this case. also use min-height instead of height here. I think you setted a specific height because since the content is different the green boxes didn't look the same, the reason for the min height is that if we add another tech with a REALLY long description the fixed height would break the layout while the min height (although makes it look horrible and you should change the min height value :P) won't break it.